wasm/module.go

```diff
@@ -35,11 +35,16 @@ type Module struct {
 	MemorySection   []*MemoryType
 	GlobalSection   []*GlobalSegment
 	ExportSection   map[string]*ExportSegment
-	StartSection    *uint32
-	ElementSection  []*ElementSegment
-	CodeSection     []*CodeSegment
-	DataSection     []*DataSegment
-	CustomSections  map[string][]byte
+	// StartSection is a function index value when set.
+	//
+	// Note: The function index space is preceded by imported functions, but not all elements in the ImportSection are
```

Comment:
fyi, this is not only true for function index space, but also table, globals memory space. According to the spec: 

> The index space for functions, tables, memories and globals includes respective imports declared in the same module. The indices of these imports precede the indices of other definitions in the same index space.

Reply:
good point! I will add the text for the others, too!